src/call.c

```diff
@@ -1606,6 +1620,15 @@ static void sipsess_estab_handler(const struct sip_msg *msg, void *arg)
 
 	call_stream_start(call, true);
 
+	/* check if replaces is supported on outgoing call */
+	hdr = sip_msg_hdr(msg, SIP_HDR_SUPPORTED);
+	if (hdr) {
+		if (!re_regex(hdr->val.p, hdr->val.l, "replaces", &v)) {
+			info("call %s supports attended transfer\n", call->id);
+			call->support_replaces = true;
+		}
+	}
+
```

Comment:
For outgoing calls we know that we support "replaces". Right?
Then you should call
```c
add_extension(ua, "replaces");
```
And initialize `support_replaces` with `true` for outgoing calls.

Reply:
I was not sure if I could add the extension replaces header for the baresip client.

If I understand https://datatracker.ietf.org/doc/html/rfc3891 correctly the client should send the header if he can send the `Replaces` header.
There will never be an call with an `Replaces` header send to the client because of it?
Will the PBX rewrite the headers if they dont support it?

Reply:
Added the configuration entry if baresip should send the replaces header.

I still think that I need to know if the telephone system I am connected to supports replaces.
If you think the config value is enough I can remove it and just check the config value.

Reply:
As @alfredh wrote. You don't need a config option. Baresip should put "replaces" into the Supported header.

In line 1619 baresip checks from incoming SIP requests if the remote supports replaces.